placementApp/views.py

```diff
@@ -1,9 +1,10 @@
-from .models import Student, Position, Company, Application
+from .models import Student, Position, Company, Application, Coordinator
 from .serializers import (
     StudentSerializer,
     PositionReadSerializer,
     PositionWriteSerializer,
     CompanySerializer,
+    CoordinatorSerializer,
 )
 import datetime
 from .serializers import *
```

Comment:
Here we have two import statements so please remove any one.

Reply:
Removed the extra import